flocker/acceptance/scripts/lsblkhttp.py

```diff
@@ -0,0 +1,42 @@
+"""
+HTTP server that returns the size of the mounted volume.
+"""
+
+from sys import argv
+from subprocess import check_output
+
+from BaseHTTPServer import BaseHTTPRequestHandler, HTTPServer
+
+
+class Handler(BaseHTTPRequestHandler):
+    def do_GET(s):
+        s.send_response(200)
+        s.end_headers()
+        if len(argv) > 1:
+            try:
+                # format output is </dev/id>[/mount/point]
+                # so we filter the mountpoint
+                # returned device is '/dev/xvdb\\n'
+                device_path = check_output(['findmnt', '-n',
+                                            '-m', '%s' % argv[1],
```

Comment:
:grey_question: This would look nicer if you moved the `'findmnt'` down to the next line and indented by 4 spaces, e.g. 

```python
device_path = check_output([
    'findmnt', '-n', '-m', '%s' % argv[1], '-o', 'SOURCE']).split(
        '[')[0].split('\n')[0]
```